tests/sentry/api/endpoints/test_organization_metric_data.py

```diff
@@ -1388,51 +1252,32 @@ def test_limit_without_orderby(self):
         within the limit, and that are also with complete data from across the entities
         """
         org_id = self.organization.id
-
-        fcp_metric = perf_indexer_record(
-            self.project.organization.id, TransactionMRI.MEASUREMENTS_FCP.value
-        )
-        tag3 = perf_indexer_record(org_id, "tag3")
-        value1 = perf_indexer_record(org_id, "value1")
-        value2 = perf_indexer_record(org_id, "value2")
-        value3 = perf_indexer_record(org_id, "value3")
-        value4 = perf_indexer_record(org_id, "value4")
-
-        self._send_buckets(
-            [
-                {
-                    "org_id": org_id,
-                    "project_id": self.project.id,
-                    "metric_id": self.session_metric,
-                    "timestamp": int(time.time()),
-                    "tags": {tag3: value1},
-                    "type": "c",
-                    "value": 10,
-                    "retention_days": 90,
-                }
-            ],
-            entity="metrics_counters",
-        )
-        self._send_buckets(
-            [
-                {
-                    "org_id": org_id,
-                    "project_id": self.project.id,
-                    "metric_id": fcp_metric,
-                    "timestamp": int(time.time()),
-                    "type": "d",
-                    "value": [1],
-                    "tags": {tag3: value},
-                    "retention_days": 90,
-                }
-                for value in (
-                    value2,
-                    value3,
-                    value4,
-                )
-            ],
-            entity="metrics_distributions",
-        )
+        self.store_metric(
```

Comment:
Why do we still need the release health data in this test?

Reply:
@ahmedetefy i think it was there even before:

https://github.com/getsentry/sentry/blob/master/tests/sentry/api/endpoints/test_organization_metric_data.py#L50

```
self.session_metric = rh_indexer_record(org_id, SessionMRI.SESSION.value)
```

I just rewrote it in more explicit way. 🤔 (without changing behavior) 

Do you want me to rewrite it for another test/metric to avoid mixing useCases here?